.circleci/config.yml

```diff
@@ -1495,7 +1495,7 @@ jobs:
           docker_layer_caching: true
       - run:
           name: Run Kontrol Tests
-          command: pnpm test:kontrol
```

Comment:
```
            curl -X POST \
              -H "Accept: application/vnd.github+json" \
              -H "Authorization: Bearer $GITHUB_TOKEN" \
              https://api.github.com/repos/runtimeverification/optimism-ci/actions/workflows/optimism-ci.yaml/dispatches \
              -d '{
                "ref": "workflow-statuses",
                "inputs": {
                  "branch_name": "develop",
                  "extra_args": "script",
                  "statuses_sha": "<< pipeline.git.revision >>",
                  "org": "runtimeverification",
                  "repository": "circleCI-TestBed",
                  "auth_token": "'"$GITHUB_TOKEN"'"
                }
              }'
 ```

Reply:
Need from Optimism to configure their circle CI Environment Variables to contain a GITHUB_TOKEN Variable with access to write commit statuses and execute workflows.

Reply:
WE need to add an additional line to fetch the XML json file and store it to packages/contracts-bedrock/<file>.json
```
      - run:
          name: Fetch Artifact Result Files from Workflow Summary in RV
          command: Curl < to fetch files from a summary> 
          working_directory: ./packages/contracts-bedrock
```